File: src/main/java/me/zeroeightsix/kami/event/events/ChunkEvent.java
package me.zeroeightsix.kami.event.events;

import me.zeroeightsix.kami.event.KamiEvent;
import net.minecraft.network.play.server.SPacketChunkData;
import net.minecraft.world.chunk.Chunk;

/**
 * @author 086
 */
public class ChunkEvent extends KamiEvent {
    private Chunk chunk;
    private SPacketChunkData packet;

    public ChunkEvent(Chunk chunk, SPacketChunkData packet) {
        this.chunk = chunk;
        this.packet = packet;
    }

    public Chunk getChunk() {
        return chunk;
    }

    public SPacketChunkData getPacket() {
        return packet;
    }
}

File: src/main/java/me/zeroeightsix/kami/event/events/ChunkEvent.kt
package me.zeroeightsix.kami.event.events

import me.zeroeightsix.kami.event.Event
import net.minecraft.network.play.server.SPacketChunkData
import net.minecraft.world.chunk.Chunk

class ChunkEvent(val chunk: Chunk, val packet: SPacketChunkData) : Event